deployment/api-deployment.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+    kompose.version: 1.22.0 (955b78124)
```

Comment:
Please remove all references to kompose

Reply:
Though it was only that one
https://github.com/change-metrics/monocle/pull/489#discussion_r675561720

Will do

Reply:
done :)